portal/plugins/gnmatomresponder/notification.py

```diff
@@ -199,6 +199,7 @@ def process_notification(notification):
     from models import ImportJob
     importjob = ImportJob.objects.get(job_id=notification.jobId)
     importjob.status = notification.status
+    importjob.processing = False
 
```

Comment:
you'll want to do `importjob.save()` here, no?

Reply:
We have it on line 207 so I assumed it was not needed twice.

Reply:
I would put it there, in case `handle_failed_job` raises an exception; in that instance the lock would not be removed.

Reply:
Okay.

Should I remove the original line with it on?

@fredex42

Reply:
no, keep it there just in case